apps/console/src/init/app-utils.ts

```diff
@@ -314,14 +320,19 @@ export const AppUtils = (function() {
 
         /**
          * Get the URL for the tenanted Myaccount.
-         *
+         * 
+         * We append any given path to a qualified path.
+         * when skipTenant is false in tenantQualifiedPath, 
+         * the argument _config.accountApp.path will get appended. 
+         * This is because through extensions we can control the MyAccount path for different deployments.
+         * 
          * @return {string}
          */
-        getTenantQualifiedAccountAppPath: function() {
-            return ((this.getTenantPrefix() !== "") && (this.getTenantName() !== "")) ?
+        getTenantQualifiedAccountAppPath: function(append) {
```

Comment:
Better if we can have a meaningful arg name here.

*suggestion*
```js
getTenantQualifiedAccountAppPath: function(pathname) {
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/3122